lib/common/components/Login.js

```diff
@@ -12,9 +12,10 @@ type Props = {
 /**
  * This component handles the `/login` route by redirecting the browser to the location specified
  * in the `redirectOnSuccess` prop (or to the previously active URL or `/home` if none is specified).
+ * FIXME: We can probably remove handling the /login route.
  */
 export default function Login ({
-  redirectOnSuccess = window.location.path || '/home'
+  redirectOnSuccess = '/home'
```

Comment:
Doesn't this remove support for returning back to a specific feed source you tried to open?